Build/build.ps1

```diff
@@ -15,22 +15,158 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
 #>
 
 <#
-  .SYNOPSIS
-  Updates the Code Coverage badge with the new percent from the unit tests.
+.SYNOPSIS
+Retrieves the Module Version from the passed psd1 file.
 
-  .PARAMETER CodeCoveragePercent
-  The new value in percents from the Code Coverage of the unit tests. Default value is 0.
+.DESCRIPTION
+Gets the content of the passed psd1 file.
+Using a regex pattern retrieves the Module Version from the content of the psd1 file.
+Returns the Module Version.
 
-  .PARAMETER TextFilePath
-  The path to the file containing the Code Coverage badge.
+.PARAMETER PsdPath
+Specifies the path to the psd1 file where the Module Version is located.
+#>
+function Get-ModuleVersion {
+    [CmdletBinding()]
+    [OutputType([string])]
+    Param(
+        [Parameter(Mandatory = $true)]
+        [string] $PsdPath
+    )
+
+    $psdFileContent = Get-Content -Path $PsdPath
+
+    $moduleVersionPattern = "(?<=ModuleVersion = ')(\d*.\d*.\d*.\d*)"
+    $moduleVersionMatch = $psdFileContent | Select-String -Pattern $moduleVersionPattern
+
+    return $moduleVersionMatch.Matches[0].Value
+}
+
+<#
+.SYNOPSIS
+Retrieves the description of the merged Pull Request via the GitHub API.
+
+.DESCRIPTION
+Constructs a query using the current commit 'sha' to retrieve the Pull Request information
+via the GitHub API using the 'Auth Token' and returns the description of the found pull request.
+The 'Auth Token' and current commit 'sha' are available through Travis CI environment variables.
+#>
+function Get-PullRequestDescription {
+    [CmdletBinding()]
+    [OutputType([string])]
+
+    $searchIssuesAPIUrl = 'https://api.github.com/search/issues'
+    $repository = 'vmware/dscr-for-vmware'
+    $searchType = 'pr'
+    $pullRequestState = 'closed'
+
+    $base64Token = [System.Convert]::ToBase64String([char[]] $env:GH_TOKEN)
+    $uri = "$searchIssuesAPIUrl?q=repo:$repository+type:$searchType+state:$pullRequestState+$env:TRAVIS_COMMIT is:merged"
```

Comment:
There is no max items limit? Is it going to return all closed pull requests?

Reply:
Well in the uri "$env:TRAVIS_COMMIT" is used so it will return only one Pull Request.